development/tools/pkg/rendertemplates/genericcomponent.go

```diff
@@ -99,3 +50,61 @@ func (r *RenderConfig) GenerateComponentJobs(global map[string]interface{}) {
 		r.Values["JobConfigs"] = r.JobConfigs
 	}
 }
+
+// generateComponentJobs generates jobs for components
+func generateComponentJobs(global map[string]interface{}, repo Repo, job Job) []Job {
+	var jobs []Job
+	// generate component jobs
```

Comment:
Did you change anything in a code or it was only moved to separate function?

Reply:
It was moved to a separate function. I've made some slight changes to split the caller function into smaller ones